phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/settings/ViewSettings.java

```diff
@@ -250,7 +251,17 @@ private void initAboutPane() {
         gridPane.add(new TextWithStyle(translate("Git Revision:"), CAPTION_TITLE_CSS_CLASS), 0, 2);
         Hyperlink gitRevisionLink = new Hyperlink(applicationGitRevision);
         gitRevisionLink.setOnAction(event ->
-                hostServicesSupplier.get().showDocument("https://github.com/PlayOnLinux/POL-POM-5/commit/" + applicationGitRevision)
+                {
+                    String url = "https://github.com/PlayOnLinux/POL-POM-5/commit/" + applicationGitRevision;
+                    // use xdg-open until HostServices works
+                    Runtime runtime = Runtime.getRuntime();
+                    try {
+                        runtime.exec("xdg-open " + url);
```

Comment:
The idea of using an interface is to be allowed to improve the implementation with corner cases later on